images/structure.md

```diff
@@ -50,15 +50,13 @@ Coder inserts static assets into each workspace, including:
 
 - code-server
 - JetBrains Projector
-- Coder CLI
+- Coder CLI, which includes the [Coder Agent](../setup/architecture.md)
```

Comment:
We also document some environment variables here: https://coder.com/docs/coder/latest/workspaces/variables#available-environment-variables - however, there are several that are deliberately undocumented because they are for internal use and may change in a future release. I suppose we could document those as well, with their purpose, as long as we also clearly indicate that they are not covered by any sort of compatibility guarantee